### PR TITLE
portfolio-setup Review

### PR DESCRIPTION
**Pull request by ANTOINE1128**
add  main folders 
add  index.html file
add image and styles.css folders

### CODE REVIEW

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @ANTOINE1128 😊,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid Code Review Request 🆘💀
You have submitted a project, that I see that doesn't have a [professional README](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by ANTOINE1128** [comment]
Hello, Adnan @AdnanOlarmmi  I fully appreciate your review.  I made the changes needed, but the request review is still pending "REVIEW PENDING".

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        </p>
+            <ul class="socialmedia">
+                <li>
+                     <a href="https://github.com/ANTOINE1128"><img src="img/github-icon.png" alt="Github"></a>
+                </li>
+             <a href="https://twitter.com/TonyMakdessy"><img src="img/tweeter-icon.png" alt="Twitter"></a>
+                </li>
+                <a href="https://www.linkedin.com/in/antoine-makdessy/"><img src="img/linkden.png" alt="linkedin"></a>
+                </li>
+            </ul>
```
Comment (osamaashraf6) [inline index.html:37]: - [ ] Kindly 😃 try to fix the li tags at the project here when I see the code I see that some li tag does not have a closing tag and some does not have an open tag and there is a conflict at the li at the ul tag, so please try to fix that for the project code that's for writing the code better without any bugs😊

| Your code| 
| --- |
|     image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styletony.css">
```
Comment (braincee) [inline index.html:7]: - [ ] Please I suggest you properly link your CSS file to the HTML file. 
In this case, your CSS file is `style.css` but in the link, it is `styletony.css`. 

```suggestion
    <link rel="stylesheet" href="style.css">
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
+.toolbar {
```
Comment (braincee) [inline style.css:1]: - [ ] Please I highly suggest adding responsive breakpoints per 
      project requirements. I am sorry it was not mentioned earlier. 😄 
       The minimum mobile screen size should be `375px`
       The minimum desktop screen size should be `768px`

- [ ] Please I highly suggest adding responsive values to your CSS 
       rules per project requirements. I am sorry it was not 
       mentioned earlier. 😄 
       Example: `width: 90%`.

This is the general and project requirements [link](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @sebalp100 ✋, I am @braincee your Code Reviewer

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - Professional README.md file ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Abenezer-Tilahun** [comment]
Hi @ANTOINE1128 😊,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid Code Review Request 🆘💀
You have submitted a project, that for the Portfolio: finish mobile version but this link is for the Portfolio: setup and mobile first  So kindly implemented the required project requirement for Portfolio: finish mobile version and submit the new pull request with all the project requirement.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Pull request merged**